diagnostic.md

```diff
@@ -5,74 +5,89 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+//  server, an application and a database. The back end is where the technical processes happen, as opposed to the front end, which is usually where the user's interaction occurs.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+// the model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+// the controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+// the controller is the view, it responds directly back to the client with the output
 ```
 
 What does C.R.U.D stand for?
 
 ```md
-// your response here
+// Create, Read, Update, Delete
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+// it is found in the controller
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```md
```

Comment:
Correspond the rails actions to the CRUD requests. 
`Index (READ), Create (CREATE), Show (READ), Update (UPDATE), Destroy (DESTROY).`